### DISCUSSION TITLE
Issue with currying?

### DISCUSSION THREAD

**Discussion by delanym** [checkstyle/checkstyle#14933] [Q&A] [answered]
Why do I get `GenericWhitespace: '>' is followed by whitespace.` with this code?

```java
    try (var executorService = Executors.newFixedThreadPool(5)) {
      IntStream.range(0, size).<Runnable> mapToObj(key -> () -> {
        cache.put(key, "value" + key);
        countDownLatch.countDown();
      }).forEach(executorService::submit);
      countDownLatch.await();
    }
```

**Comment by strkkk** [comment, marked as answer]
You get this because you have a whitespace after `<Runnable>`
`IntStream.range(0, size).<Runnable>mapToObj(key -> () -> {` wont cause violation

**Reply by delanym** [reply]
Found the culprit `<setting id="org.eclipse.jdt.core.formatter.insert_space_after_closing_angle_bracket_in_type_arguments" value="insert"/>`
